frontend/packages/core/src/Feedback/alert.tsx

```diff
@@ -76,14 +85,37 @@ const iconMappings = {
 export interface AlertProps
   extends Pick<MuiAlertProps, "severity" | "action" | "onClose" | "elevation" | "variant"> {
   title?: React.ReactNode;
+  collapsible?: boolean;
+  defaultOpen?: boolean;
+  hover?: boolean;
 }
 
-export const Alert: React.FC<AlertProps> = ({ severity = "info", title, children, ...props }) => (
-  <StyledAlert severity={severity} iconMapping={iconMappings} {...props}>
-    {title && <AlertTitle>{title}</AlertTitle>}
-    {children}
-  </StyledAlert>
-);
+export const Alert: React.FC<AlertProps> = ({
```

Comment:
i would make a separate collapsible alert component.

because collapsible could mean different things, we may want an alert later that collapses vertically (i.e. displays title with collapsible long text).

also because  the `hover` property does nothing if `collapsible` is false.

Reply:
Yeah that was my initial plan before I went this way, let me make it separate and clean it up